chucks-pick3-mobile/src/app/components/generate-next-numbers-card/generate-next-numbers-card.component.ts

```diff
@@ -99,8 +104,10 @@ export class GenerateNextNumbersCardComponent implements OnInit {
     public submitGenerate(): void {
         this.changeNavigation(1);
         this.replaceGeneratedNumbers();
-        console.log(this.drawTimeService.viewPicksArray);
-        /*this.router.navigate(['/view-picks']);*/
+        this.changeNavigation(4);
```

Comment:
Use Enums for navigation instead of numbers.  Make this change all through out the app.

Reply:
Resolved with enums